src/MaxText/input_pipeline/_hf_data_processing.py

```diff
@@ -56,6 +56,7 @@ def vision_sft_preprocessing_pipeline(
             "max_num_images_per_example": config.max_num_images_per_example,
         },
         remove_columns=image_column,  # Drop the original image columns
+        image_column="images",
```

Comment:
I believe this line needs to go after the dataset.map call within the if statement. Line 61 in this file